free-programming-books.md

```diff
@@ -237,7 +237,7 @@
 * [Learn OpenGL](http://learnopengl.com) - Joey de Vries
 * [Learning Modern 3D Graphics Programming](https://web.archive.org/web/20150225192611/http://www.arcsynthesis.org/gltut/index.html) (draft)
 * [OpenGL](http://www.songho.ca/opengl/index.html) - Concepts and illustrations.
-* [OpenGL Programming Guide (The Red Book)](http://fly.srk.fer.hr/~unreal/theredbook/)
+* [OpenGL Programming Guide (The Red Book)](https://sgar91.files.wordpress.com/2010/12/opengl-programming-guide-7e.pdf)
```

Comment:
I think this is the line Travis build is complaining about. Needs PDF indicator.